front/app/containers/Admin/reporting/components/ReportBuilderPage/ReportRow/Buttons/ShareReportButton.tsx

```diff
@@ -26,6 +29,7 @@ const ShareReportButton = ({ reportId, buttonStyle = 'secondary' }: Props) => {
       <Button
         icon="share"
         buttonStyle={buttonStyle}
+        bgColor={colors.primary}
```

Comment:
The text is not very readable now 
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/91883434/bb508b92-6de4-4ef8-a229-3847a58b3cc4)

Reply:
Ah nice catch! Forgot the button is also used there.